docs/base-api.md

```diff
@@ -1436,22 +1436,12 @@ $baseApi->deletePullZone(
 );
 ```
 
-#### [Delete Edge Rule](https://docs.bunny.net/reference/pullzonepublic_deleteedgerule)
-
-```php
-$baseApi->deleteEdgeRule(
-    pullZoneId: 1,
-    edgeRuleId: 'c71d9594-3bc6-4639-9896-ba3e96217587',
-);
-```
-
-#### [Add/Update Edge Rule](https://docs.bunny.net/reference/pullzonepublic_addedgerule)
+#### [Add Edge Rule](https://docs.bunny.net/reference/pullzonepublic_addedgerule)
 
 ```php
-$baseApi->addOrUpdateEdgeRule(
+$baseApi->addEdgeRule(
     pullZoneId: 1,
     body: [
-        'Guid' => 'c71d9594-3bc6-4639-9896-ba3e96217587',
```

Comment:
Note:

This has been removed here as the `Guid` is only needed for updating an edge rule. When adding a new edge rule, a `Guid` will be created for the new edge rule and will be available in the response.